pkg/defaults/defaults.go

```diff
@@ -387,8 +387,12 @@ func stepConfigsForBuild(config *api.ReleaseBuildConfiguration, jobSpec *api.Job
 		}
 	}
 
-	for i := range config.Tests {
-		buildSteps = append(buildSteps, api.StepConfiguration{TestStepConfiguration: &config.Tests[i]})
+	for _, test := range config.Tests {
+		// TODO remove when the migration is completed
```

Comment:
Instead of ignoring it, you should set the stage for someone to use the new format. So

```go
if config[i].ContainerTestConfiguration != nil  && len(config[i].From) == 0 {
    config[i].From = config[i].ContainerTestConfiguration.From
}
```

Also add validation to ensure that `config[i].ContainerTestConfiguration == nil` XOR `len(config[i].From) == 0`

Reply:
My thought was to do it in a separate PR, but it doesn't really matter. I'll add it here.

Reply:
No preference on my part -- let's do it here

Reply:
Changed.